### PR TITLE
Add PygmalionAI-PIPPA Dataset

### PR DESCRIPTION
**Pull request by the-infiltrator**

### CODE REVIEW

File: constants/creator_groups.json [hunk 1]
```diff
@@ -343,7 +343,8 @@
         "Salesforce Research",
         "Samsung R&D Institute",
         "Tencent AI Lab",
-        "iFLYTEK Research"
+        "iFLYTEK Research",
```
Comment (shayne-longpre) [inline constants/creator_groups.json:346]: Could you put these in alphabetical order -- not sure why iFLYTEK was out of order. Thanks!

File: data_summaries/PygmalionAI-PIPPA.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            "null"
+        ],
+        "Creators": [
+            "PygmalionAI"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://www.apache.org/licenses/LICENSE-2.0"
+            }
```
Comment (shayne-longpre) [inline data_summaries/PygmalionAI-PIPPA.json:41]: We may need to add the Character AI terms as well (to here and to the license classes file). cc @rmahari to interpret their terms: https://character.ai/tos

File: data_summaries/PygmalionAI-PIPPA.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        ],
+        "Model Generated": [
+            "Other"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "null"
```
Comment (shayne-longpre) [inline data_summaries/PygmalionAI-PIPPA.json:32]: empty list, not ["null"]

File: data_summaries/PygmalionAI-PIPPA.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        ],
+        "Task Categories": [
+            "Conversational Question Answering",
+            "Roleplay"
+        ],
+        "Text Sources": [
+            "https://character.ai"
+        ],
+        "Model Generated": [
+            "Other"
```
Comment (shayne-longpre) [inline data_summaries/PygmalionAI-PIPPA.json:25]: "Character AI"

File: data_summaries/PygmalionAI-PIPPA.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        "ArXiv URL": "https://arxiv.org/abs/2308.05884",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "PygmalionAI-PIPPA",
+        "Collection URL": "https://huggingface.co/PygmalionAI",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Conversational Question Answering",
+            "Roleplay"
```
Comment (shayne-longpre) [inline data_summaries/PygmalionAI-PIPPA.json:19]: Dialog as well

File: data_summaries/PygmalionAI-PIPPA.json [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            "PygmalionAI"
+        ],
+        "Licenses": [
+            {
+                "License": "Apache License 2.0",
+                "License URL": "https://www.apache.org/licenses/LICENSE-2.0"
+            }
+        ],
+        "License Notes": "PIPPA contains NSFW and heavily disturbing content. Models trained purely with PIPPA may generate X-rated output.",
+        "License Verified By": "",
```
Comment (shayne-longpre) [inline data_summaries/PygmalionAI-PIPPA.json:44]: your name :)

**Review by shayne-longpre — APPROVED** [review]
Looks good to me after:
- confirm the license/terms with Robert
- make minor fixes
- check outputs look correct
- make sure it passes all tests without any warnings

File: constants/creator_groups.json (hunk 1 above)
Comment (the-infiltrator) [reply constants/creator_groups.json]: Fixed in latest commit.

File: data_summaries/PygmalionAI-PIPPA.json (hunk 2 above)
Comment (the-infiltrator) [reply data_summaries/PygmalionAI-PIPPA.json:41]: This is pending. Can you provide information on what would be most appropriate here?

File: data_summaries/PygmalionAI-PIPPA.json (hunk 6 above)
Comment (the-infiltrator) [reply data_summaries/PygmalionAI-PIPPA.json:44]: Resolved.

File: data_summaries/PygmalionAI-PIPPA.json (hunk 5 above)
Comment (the-infiltrator) [reply data_summaries/PygmalionAI-PIPPA.json:19]: Resolved

File: data_summaries/PygmalionAI-PIPPA.json (hunk 2 above)
Comment (rmahari) [reply data_summaries/PygmalionAI-PIPPA.json:41]: All, 0, 0

**Pull request merged**